docs/internal_api/index.rst

```diff
@@ -104,12 +104,20 @@ Connectivity
 .. autosummary::
    :toctree: _autosummary
 
+
+
+
    grid.connectivity._face_nodes_to_sparse_matrix
    grid.connectivity._replace_fill_values
    grid.connectivity._build_nNodes_per_face
    grid.connectivity._build_edge_node_connectivity
    grid.connectivity._build_face_edges_connectivity
    grid.connectivity._build_node_faces_connectivity
+   grid.connectivity._build_edge_face_connectivity
+   grid.connectivity._populate_edge_node_connectivity
+   grid.connectivity._populate_face_edges_connectivity
+   grid.connectivity._populate_node_faces_connectivity
+   grid.connectivity._populate_edge_face_connectivity
```

Comment:
Looks like we need to add `_populate_n_nodes_per_face` as well into this list.

Reply:
Good catch!